blocks/inspector-controls/select-control/index.js

```diff
@@ -14,30 +14,33 @@ import { withInstanceId } from '@wordpress/components';
 import BaseControl from './../base-control';
 import './style.scss';
 
-function SelectControl( { label, selected, instanceId, onBlur, options = [], ...props } ) {
+function SelectControl( { label, instanceId, onChange, options = [], ...props } ) {
 	const id = 'inspector-select-control-' + instanceId;
-	const onBlurValue = ( event ) => onBlur( event.target.value );
+	const onChangeValue = ( event ) => onChange( event.target.value );
 
+	// Disable reason: A select with an onchange throws a warning
```

Comment:
This isn't so much a reason as it is stating a fact. The rule exists because `onChange` and `onBlur` have accessibility impacts when used in `select` elements. See also #2227